Description: Correct pod NAME issues in LedgerSMB/DBObject/Company.pm
 Has the module file name but does not have the dash plus description in the
 NAME entry.
Bug: http://sourceforge.net/tracker/?func=detail&aid=3472556&group_id=175965&atid=1005191
Author: Robert James Clay <jame@rocasa.us>

--- a/LedgerSMB/DBObject/Company.pm
+++ b/LedgerSMB/DBObject/Company.pm
@@ -1,6 +1,7 @@
 =head1 NAME
 
-LedgerSMB::DBObject::Company.pm
+LedgerSMB::DBObject::Company - Contains the base utility functions for creating,
+                            saving, and retrieving customers and vendors.
 
 =head1 SYNOPSIS
 
